dotcom-rendering/src/web/components/ArticleMeta.tsx

```diff
@@ -28,6 +30,9 @@ type Props = {
 	primaryDateline: string;
 	secondaryDateline: string;
 	branding?: Branding;
+	discussionApiUrl: string;
+	shortUrlId: string;
+	isCommentable: boolean;
```

Comment:
Here we're prop drilling these values down so we can serialise them in the Island